### PR TITLE
[KOGITO-5389] Add "Asking for Help" section to README

### PR DESCRIPTION
**Pull request by Kevin-Mok**
See: https://issues.redhat.com/browse/KOGITO-5389

## Description
With the "Issues" section of this repository being closed soon (see @ricardozanini's email in the BSIG mailing list about this), users should be directed to our other Kogito communities to ask for help.

## Reducing README Whitespace
I also removed unnecessary line breaks and changed the headers in the README to use the `h2` Markdown tag instead of a horizontal line. It produces the same effect - see my test README [here](https://github.com/Kevin-Mok/kogito-runtimes/tree/KOGITO-5389-headers#kogito) where the first header is the `h2` Markdown tag and the second is the original header with the horizontal line. In total, these changes don't modify the resulting HTML README, but they make the raw Markdown README cleaner. This breaks the "Pull Request does not include fixes for issues other than the main ticket" PR requirement, but this also doesn't really seem like it's worth opening another Jira for. :stuck_out_tongue: 

## Remaining README's to Replicate this PR To
- [x] [kogito-examples](https://github.com/kiegroup/kogito-examples/pull/756)
- [x] [kogito-apps](https://github.com/kiegroup/kogito-apps/pull/916)

## Requirements
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/master/kogito-build/kogito-ide-config)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run all builds</b>  
  Please add comment: <b>Jenkins retest this</b>

* <b>Run (or rerun) specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] tests</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Run (or rerun) LTS specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Run (or rerun) native specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
jenkins retest optaplanner

**Comment by ricardozanini** [comment]
jenkins rerun optaplanner

**Comment by ricardozanini** [comment]
Jenkins run optaplanner tests

**Comment by ricardozanini** [comment]
Jenkins run apps tests

**Review by cristianonicolai — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,10 +37,8 @@ If you want to read more about Quarkus:
...
-
-
-Building and Contributing to Kogito
------------------------------------
-
+## Building and Contributing to Kogito
 All contributions are welcome! Before you start please read the [contribution guide](CONTRIBUTING.md).
 
+## Asking for Help
+If you have any questions about the project or are facing any problems running/deploying our code, please open a new thread in the [Kogito stream](https://kie.zulipchat.com/#narrow/stream/232676-kogito) of the KIE Zulip chat, or send an email to the [Kogito mailing list](https://groups.google.com/g/kogito-development). If you would like to report a bug or have a feature request, please feel free to open a Jira for it in the [Kogito project](https://issues.redhat.com/projects/KOGITO/summary).
```
Comment (danielezonca) [inline README.md:44]: I would suggest to change the order:
- have you found a problem and you have a small project to reproduce it? Open a Jira
- are you facing some problem and you don't really know what is going on? Use the mailing list
- don't you like mailing list or do you want to discuss about a feature/enhancement? Open a new thread in our chat

Wdyt?

Comment (mareknovotny) [reply README.md:44]: I think differentiate between mailing list and Google Groups ;) 
And I would not use wording 
> 
>     * don't you like mailing list or do you want to discuss about a feature/enhancement? Open a new thread in our chat

instead 
If you need to chat about your problem or discuss about a feature/enhancement? Open a new thread in our Zulip Chat.

Comment (MarianMacik) [reply README.md:44]: +1 for a different order and stating that it is actually a Google Group.

**Review by MarianMacik — APPROVED** [review]
I agree with Daniele's and Marek's proposals.

File: README.md (hunk 1 above)
Comment (Kevin-Mok) [reply README.md]: I revised the section to organize the communities for specific use cases as @danielezonca suggested, followed the same ordering and corrected the mailing list to be a Google Group. Please let me know what you think of the formatting and questions/answers.

**Comment by ricardozanini** [comment]
Wow! Much better now! Thanks for the feedback, guys!

**Review by danielezonca — APPROVED** [review]
Thanks for the changes 👍

**Comment by ricardozanini** [comment]
@Kevin-Mok I think we are good to go. I'll merge this one. Can you also please open the PR for the remaining repos?

**Comment by Kevin-Mok** [comment]
Oops, that was an accidental push. Let me revert that quickly.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**